config/samples/mellanox.com_v1alpha1_nicclusterpolicy.yaml

```diff
@@ -19,7 +19,7 @@ spec:
   ofedDriver:
     image: doca-driver
     repository: nvcr.io/nvstaging/mellanox
-    version: 24.01-0.3.3.1
+    version: 24.01-0.3.3.1.3
```

Comment:
is this version available in ngc ?

is it a new version format ?

Reply:
Image is available in NGC for me - pulling works. It's not available in /auto/mswg/... though - so this will break the precompiled CI fix. If it's a new version format we might need to do more work to support it

Reply:
yes, this build is available. We use this format when image build is different from driver build